.editorconfig

```diff
@@ -10,3 +10,7 @@ indent_style = space
 indent_size = 4
 insert_final_newline = true
 trim_trailing_whitespace = true
+
+[*.yml]
+indent_style = space
+indent_size = 2
```

Comment:
Is there a reason for repeating the `indent_style`?

Also, why not using something like `[*.{json,yml}]` since I believe it applies to both?